collector/info_schema_innodb_cmp.go

```diff
@@ -0,0 +1,98 @@
+// Scrape `information_schema.INNODB_CMP`.
+
+package collector
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	//	"github.com/prometheus/common/log"
+)
+
+const innodbCmpQuery = `SELECT * FROM information_schema.INNODB_CMP`
+
+var (
+	// Map known client-statistics values to types. Unknown types will be mapped as
+	// untyped.
+	informationSchemaInnodbCmpTypes = map[string]struct {
```

Comment:
It's not necessary to use a struct like this.  Take a look at the [innodb_metrics](https://github.com/prometheus/mysqld_exporter/blob/master/collector/info_schema_innodb_metrics.go#L21-L43) for a good example.

This code isn't wrong, but I would like to keep a consistent simple style between all of the collectors.  Because we prefer explicit column names, the code is simplified.

Reply:
Code was rewritten as in the pointed example.